Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -611,6 +611,51 @@ protected override void ContextMenuClosed()
         {
             _buttonController.RemoveFixed();
         }
+
+        protected override void OnPaint(PaintEventArgs e)
+        {
+            if (DialogResult == DialogResult.None)
+            {
```

Comment:
What ?
When does this happen,
Always..
What about user text already applied to the button ?

Reply:
Can put in a temporary variable to save the current text. The `None` will fire **only** if the dev had previously changed its `DialogResult` state, but then changed it back  to `None`.

Reply:
Or... to remove the `None` block, as KM does not have a `GlobalString` for `DialogResult.None`?